tests/puppeteer/campaigns/data/demo/categories.js

```diff
@@ -1,7 +1,32 @@
 module.exports = {
   Categories: {
-    accessories: {id: '6', name: 'Accessories'},
-    stationery: {id: '7', name: 'Stationery'},
-    men: {id: '4', name: 'Men'},
+    art: {
+      id: '9',
+      name: 'art',
+      description: 'Framed poster and vector images',
+      position: '3',
+      displayed: 'Yes',
```

Comment:
Consider this kind of property as boolean
```suggestion
      displayed: true,
```